## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 465066`
Commit: [`69394c3b5d60b8cc5ae27c4c8234f6ed11e2e2d1`](https://github.com/NixOS/nixpkgs/commit/69394c3b5d60b8cc5ae27c4c8234f6ed11e2e2d1) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/69394c3b5d60b8cc5ae27c4c8234f6ed11e2e2d1..pull/465066/head))
Merge: [`914c98d89fc4f15f6ed3518172a3ad7b8c4b66a2`](https://github.com/NixOS/nixpkgs/commit/914c98d89fc4f15f6ed3518172a3ad7b8c4b66a2)

Logs: https://github.com/Aleksanaa/nixpkgs-review-gha/actions/runs/19761036994


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>openboardview</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>openboardview</li>
  </ul>
</details>

---
### `x86_64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>openboardview</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>openboardview</summary>
<pre>  407 |   _LIBCPP_HIDE_FROM_ABI path(const path&amp; __p) : __pn_(__p.__pn_) {}
      |                         ^
/nix/build/nix-8486-1004395974/source/src/openboardview/confparse.cpp:508:12: error: &#x27;~path&#x27; is unavailable: introduced in macOS 10.15
  508 |                                                 auto nfn = filepath;
      |                                                      ^
/nix/store/1mzb0lbliqa5spskrmwj6qxv2gpi9gwx-libcxx-19.1.2+apple-sdk-15.5/include/c++/v1/__filesystem/path.h:434:25: note: &#x27;~path&#x27; has been explicitly marked unavailable here
  434 |   _LIBCPP_HIDE_FROM_ABI ~path() = default;
      |                         ^
/nix/build/nix-8486-1004395974/source/src/openboardview/confparse.cpp:509:11: error: &#x27;operator+=&#x27; is unavailable: introduced in macOS 10.15
  509 |                                                 nfn += &quot;~&quot;;
      |                                                     ^
/nix/store/1mzb0lbliqa5spskrmwj6qxv2gpi9gwx-libcxx-19.1.2+apple-sdk-15.5/include/c++/v1/__filesystem/path.h:575:31: note: &#x27;operator+=&#x27; has been explicitly marked unavailable here
  575 |   _LIBCPP_HIDE_FROM_ABI path&amp; operator+=(const value_type* __x) {
      |                               ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make[2]: *** [src/openboardview/CMakeFiles/openboardview.dir/build.make:102: src/openboardview/CMakeFiles/openboardview.dir/confparse.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:364: src/openboardview/CMakeFiles/openboardview.dir/all] Error 2
make: *** [Makefile:156: all] Error 2</pre>
</details>
</details>

---
### `aarch64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>openboardview</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>openboardview</summary>
<pre>  407 |   _LIBCPP_HIDE_FROM_ABI path(const path&amp; __p) : __pn_(__p.__pn_) {}
      |                         ^
/nix/build/nix-9235-250239248/source/src/openboardview/confparse.cpp:508:12: error: &#x27;~path&#x27; is unavailable: introduced in macOS 10.15
  508 |                                                 auto nfn = filepath;
      |                                                      ^
/nix/store/1a05xixrdmk175drdln3w047x3dxfr7l-libcxx-19.1.2+apple-sdk-15.5/include/c++/v1/__filesystem/path.h:434:25: note: &#x27;~path&#x27; has been explicitly marked unavailable here
  434 |   _LIBCPP_HIDE_FROM_ABI ~path() = default;
      |                         ^
/nix/build/nix-9235-250239248/source/src/openboardview/confparse.cpp:509:11: error: &#x27;operator+=&#x27; is unavailable: introduced in macOS 10.15
  509 |                                                 nfn += &quot;~&quot;;
      |                                                     ^
/nix/store/1a05xixrdmk175drdln3w047x3dxfr7l-libcxx-19.1.2+apple-sdk-15.5/include/c++/v1/__filesystem/path.h:575:31: note: &#x27;operator+=&#x27; has been explicitly marked unavailable here
  575 |   _LIBCPP_HIDE_FROM_ABI path&amp; operator+=(const value_type* __x) {
      |                               ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make[2]: *** [src/openboardview/CMakeFiles/openboardview.dir/build.make:102: src/openboardview/CMakeFiles/openboardview.dir/confparse.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:364: src/openboardview/CMakeFiles/openboardview.dir/all] Error 2
make: *** [Makefile:156: all] Error 2</pre>
</details>
</details>

--- a/pkgs/by-name/op/openboardview/package.nix
+++ b/pkgs/by-name/op/openboardview/package.nix
@@ -15,22 +15,21 @@
 
 stdenv.mkDerivation rec {
   pname = "openboardview";
-  version = "9.95.0";
+  version = "9.95.2";
 
   src = fetchFromGitHub {
     owner = "OpenBoardView";
     repo = "OpenBoardView";
     tag = version;
-    hash = "sha256-sKDDOPpCagk7rBRlMlZhx+RYYbtoLzJsrnL8qKZMKW8=";
+    hash = "sha256-B5VnuycRt8h7Cz3FTIbhcGcXuA60zPCz0FMvFENTwws=";
     fetchSubmodules = true;
   };
 
   patches = [
-    # Fix gcc-13 build failure
     (fetchpatch {
-      name = "gcc-13.patch";
-      url = "https://github.com/OpenBoardView/OpenBoardView/commit/b03d0f69ec1611f5eb93f81291b4ba8c58cd29eb.patch";
-      hash = "sha256-Hp7KgzulPC2bPtRsd6HJrTLu0oVoQEoBHl0p2DcOLQw=";
+      name = "fix-darwin-build.patch";
+      url = "https://github.com/OpenBoardView/OpenBoardView/commit/a1de2e5de908afd83eceed757260f6425314af2e.patch?full_index=1";
+      hash = "sha256-DK+K4F0+QGqaoWCyc8AvuIsaiTCqhAG6AsTNg2hegh0=";
     })
   ];
 

